src/internal/group_api/mod.rs

```diff
@@ -140,12 +140,14 @@ impl GroupMetaResult {
     }
 }
 
+#[derive(Debug)]
```

Comment:
Going forward we want all *Result types to be Debug, Clone, PartialEq. See #58 
```suggestion
#[derive(Debug, Clone, PartialEq)]
```